airflow/providers/docker/operators/docker.py

```diff
@@ -276,7 +277,10 @@ def __init__(
         self.log_opts_max_size = log_opts_max_size
         self.log_opts_max_file = log_opts_max_file
         self.ipc_mode = ipc_mode
-        self.skip_exit_code = skip_exit_code
+        if skip_exit_code:
```

Comment:
Is `skip_exit_code=0` a valid use case? If it is, we’ll need to use `is not None` here.

Reply:
I'll add it just in case